### PR TITLE
[JENKINS-66887] Enable jdk11 build on CI for linux and windows

### PR DESCRIPTION
**Pull request by aHenryJard**
`recommendedConfigurations` is not building jdk11 anymore. Adding it explicitly.

- [ ] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [ ] Ensure that the pull request title represents the desired changelog entry
- [ ] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

**Review by Vlatombe — APPROVED** [review]

**Review by batmat — APPROVED** [review]
Given I'm a declared maintainer/releaser in RPU, I'm going to go ahead and merge this.

I will however follow the result of the build once merged, given currently it will not actually have used the Jenkinsfile changes put here as Angélique is not a maintainer.

**Comment by batmat** [comment]
Good news, it also succeeded after merging 
image

**Pull request merged**
